apps/directory/src/components/filters/OntologyFilter.vue

```diff
@@ -22,11 +22,12 @@
     <div class="ontology pt-3 d-flex justify-content-center">
       <template v-for="ontologyId of ontologyIdentifiers" :key="ontologyId">
         <div v-show="selectedOntology === ontologyId">
-          <spinner class="mt-4 mb-5" v-if="!ontologyOptions" />
+          <spinner class="mt-4 mb-5" v-if="!resolvedOptions" />
           <tree-component
             v-else-if="displayOptions.length"
             :options="displayOptions"
             :filter="ontologyQuery"
+            :facetIdentifier="facetIdentifier"
```

Comment:
not having this property here caused console errors

Reply:
This fixes an issue where selecting a diagnosis filter throws  `this.filterInfoDictionary[t]` is undefined in the console